packages/react-tinacms-github/src/form/useGithubJsonForm.ts

```diff
@@ -0,0 +1,48 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import { FormOptions, usePlugins, Field } from 'tinacms'
+import { GitFile } from './useGitFileSha'
+import { GithubOptions } from './GithubOptions'
+import { useGithubFileForm } from './useGithubFileForm'
+
+interface Options {
+  id?: string
+  label?: string
+  fields?: Field[]
+  actions?: FormOptions<any>['actions']
+}
+
+const serialize = (formData: any) => {
+  return JSON.stringify(formData, null, 2)
+}
+
+export function useGithubJsonForm(
+  jsonFile: GitFile,
+  formOptions: Options,
+  githubOptions: GithubOptions
+) {
+  const [values, form] = useGithubFileForm(
+    jsonFile,
+    formOptions,
+    githubOptions,
+    serialize
+  )
+  usePlugins(form as any)
```

Comment:
I think this call to `usePlugins` is redundant, since `useGithubFileForm` calls `useLocalForm` which already adds the plugin.

so this could be simplified to `return useGithubFileForm(...)`